src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
```

Comment:
Can be simplified 
const women = people.filter((p) =>
    p.sex === 'f' && (!withChildren || people.some((p1) => p1.mother === p.name))
  );